docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     networks:
       - app
     ports:
-      - '3306'
+      - "3306"
```

Comment:
We should not change this to "" as it don't match when updating later on with templates from itkdev docker setup

Reply:
sorry, my linter